New Resource: `azurerm_mobile_network_slice`; New Data Source: `azurerm_mobile_network_slice` (#20336)
hashicorp/terraform-provider-azurerm: website/docs/d/mobile_network_slice.html.markdown

---
subcategory: "Mobile Network"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_mobile_network_slice"
description: |-
  Get information about a Mobile Network Slice.
---

# azurerm_mobile_network_slice

Get information about a Mobile Network Slice.

## Example Usage

```hcl
data "azurerm_mobile_network" "example" {
  name                = "example-mn"
  resource_group_name = "example-rg"
}

data "azurerm_mobile_network_slice" "example" {
  name              = "example-mns"
  mobile_network_id = data.azurerm_mobile_network.test.id
}
```

## Arguments Reference

The following arguments are supported:

> * `name` - (Required) Specifies the name which should be used for this Mobile Network Slice. Changing this forces a new Mobile Network Slice to be created.
> 
> * `mobile_network_id` - (Required) The ID of Mobile Network which the Mobile Network Slice belongs to. Changing this forces a new Mobile Network Slice to be created.

Reviewer: We can omit the force new warnings here given that this is the datasource

```suggestion
* `name` - (Required) Specifies the name which should be used for this Mobile Network Slice.

* `mobile_network_id` - (Required) The ID of Mobile Network which the Mobile Network Slice belongs to. 
```

## Attributes Reference

In addition to the Arguments listed above - the following Attributes are exported:

* `id` - The ID of the Mobile Network Slice.

> * `location` - The Azure Region where the Mobile Network Slice should exist. Changing this forces a new Mobile Network Slice to be created.

Reviewer: ```suggestion
* `location` - The Azure Region where the Mobile Network Slice should exist. 
```

* `single_network_slice_selection_assistance_information` - A `single_network_slice_selection_assistance_information` block as defined below. Single-network slice selection assistance information (S-NSSAI). Unique at the scope of a mobile network.

> * `description` - An optional description for this network slice.

Reviewer: ```suggestion
* `description` - A description of this Mobile Network Slice.
```

> * `tags` - A mapping of tags which should be assigned to the Mobile Network Slice.

Reviewer: ```suggestion
* `tags` - A mapping of tags which are assigned to the Mobile Network Slice.
```
